test/Token.t.sol

```diff
@@ -533,4 +533,126 @@ contract TokenTest is NounsBuilderTest, TokenTypesV1 {
         assertEq(mintedTokens[f2Wallet], f2Percentage);
         assertEq(mintedTokens[f3Wallet], f3Percentage);
     }
+
+    function test_updateMinters() public {
```

Comment:
Also see above – this test is already decently hardened but let's just be extra extra careful here and make this a fuzz test as well. 

```solidity
function test_updateMinters(address m1, address m2, bool m1CanMint, bool m2CanMint) public {}
```